azure-functions/acbs-function/mappings/facility/facility-loan-amend.js

```diff
@@ -7,7 +7,7 @@ const { FACILITY } = CONSTANTS;
 const facilityLoanAmend = (amendments, facility, facilityMasterRecord) => {
   try {
     // Default facility loan record
-    let record = { portfolioIdentifier: FACILITY.PORTFOLIO.E1 };
+    let record = {};
```

Comment:
are we not using this anymore? i..e default value?

Reply:
Nope this is being inserted by APIM.